src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -67,3 +67,4 @@ notification.margin.saved.title=Позиция Стикера сохранена
 settings.general.sticker.reset.margins=Сбросить отступы Стикера
 settings.general.content.sticker.margin.info=Кликните два раза по стикерам, чтобы сохранить текущую позицию.
 settings.general.content.sticker.ignore.scaling=Игнорировать масштабирование
+settings.general.randomizer.install=Randomly pick theme
```

Comment:
@Dragon-0609 I added another one of these that need to be translated. Feel free to make another PR, if this one is already merged. Thanks!